tests/runtests.sh

```diff
@@ -158,7 +173,7 @@ seleniumname=seleniumserver-$id
 
 toasterlog=$tempdir/toaster.log
```

Comment:
Because the sentinels for the containers are in these two files, local mode will not work twice in a row. The other works because everything is in a tempdir that gets created each time? Do you not want to do that in local mode because of speed?

If that's the case then the two files are going to have to be deleted if running in local mode. Although I prefer still using a temporary directory.